keg_bouncer/db/__init__.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import absolute_import
+
+import sqlalchemy as sa
+import sqlalchemy.orm as saorm
+from keg.db import db
+from keg_elements.db.mixins import MethodsMixin
+
+
+class Permission(db.Model, MethodsMixin):
```

Comment:
For consistency in our ecosystem, I'd like to see `.model.entities`.  Although, I wonder if just `.model` might be appropriate in some situations in the future.  But let's now have that debate now.  :)